scripts/publish.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+BUILD_NR=${1}
+
+REPO_DIR="$(cd "$(dirname "$0")/../" && pwd)"
+
+# Create Version
+git fetch --force --tags
+TAG=$(git tag --sort=-version:refname | head -1)
+
+if [[ "${IS_RELEASE}" == "true" ]]; then
```

Comment:
```suggestion
if [[ "${IS_RELEASE:-false}" == "true" ]]; then
```

if you take my suggestion by using `set -u` above, you'll need this. it makes the string default to "false" if `IS_RELEASE` is undefined.

alternatively you can do just `${IS_RELEASE:-}` and it will default to an empty string.